paymentsheet/res/values/strings.xml

```diff
@@ -1,11 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <resources xmlns:tools="http://schemas.android.com/tools">
-  <!-- Use when a required field has not been filled in -->
-  <string name="incomplete" tools:ignore="MissingTranslation">%s is incomplete.</string>
-  <!-- Use when a required field is not valid and cannot be corrected. -->
-  <string name="malformed" tools:ignore="MissingTranslation">%s is malformed.</string>
   <!-- Use when a required field is blank and submit is clicked. -->
-  <string name="blank_and_required" tools:ignore="MissingTranslation">%s cannot be blank.</string>
+  <string name="blank_and_required" tools:ignore="MissingTranslation">This field cannot be blank.</string>
```

Comment:
Because this will not necessarily be under a particular single field (as in the case of billing or other such things).   It might be better for this to say: "There is a black field that is required" or something like that.